src/views/Player.vue

```diff
@@ -97,7 +100,8 @@ export default {
           process.env.VUE_APP_BACKEND +
             process.env.VUE_APP_BACKEND_PLIO_DETAILS +
             "?plioId=" +
-            this.$route.params.id
+            this.$route.params.id +
```

Comment:
oh.... this is needed urgently no? so is okay. We can merge.

Because getting video details shouldn't depend on user ID (except for authentication). Abhi ke liye merge kar lo. Let me think a bit. With current JSONs this maybe fine, but in near future, we will change in some way.

Just to keep in mind

Reply:
Yep. And the `userId` param is optional in the backend code - One can send in the `userId` here as well to get the session ID - or else if it is not passed - sessionId will be 1. But I agree, this can be a different API - separate from the one supposed to only get the plio details.